.github/workflows/deploy-pr.yml

```diff
@@ -104,15 +104,15 @@ jobs:
           git config user.name "github-actions[bot]"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git add ./${{ steps.build-info.outputs.deploy_path }}
-          git commit -m "build ${{ steps.build-info.outputs.deploy_path }} ${{ steps.build-info.outputs.repo_name }}@${{ steps.build-info.outputs.ref }}"
+          git commit -m "build ${{ steps.build-info.outputs.deploy_path }} ${{ steps.build-info.outputs.repo_name }}@$(cd .. && git rev-parse HEAD)"
```

Comment:
What's the difference?

Reply:
`git rev-parse HEAD` is a SHA like `d3c1dc6940d0e51404ffa562e4e6b8d2e2a1e799`
`steps.build-info.outputs.ref`  is the branch name `upopt`